### DISCUSSION TITLE
duda ejercicio 7 tp1

### DISCUSSION THREAD

**Discussion by Juampi44** [INGCOM-UNRN-P1/cursada-2023#81] [Preguntas y respuestas] [answered]
me corrigieron lo siguiente: 'Cambiá por un lazo de bandera'. de la linea
```C
if (numero % i == 0)
{
  es_primo = 0;
  break;
```
a que se refiere con lazo de bandera?

**Comment by martinvilu** [comment, marked as answer]
Fue explicado en clases, es otra forma de armar lazos en donde la condición incluye una variable booleana.

**Reply by martinvilu** [reply]
Algo como
```C
while (es_primo && (numero % i == 0))
```
Por ejemplo
